### PR TITLE
DF-325-Removing redundant topics from the docs website PEM online help

### PR DESCRIPTION
**Pull request by arup-roy2023**
## What Changed?
Removing redundant links in the Docs website online help and adding redirects in the doc topic files

### CODE REVIEW

File: product_docs/docs/pem/8/managing_pem_agent.mdx [hunk 1]
```diff
@@ -8,6 +8,7 @@ legacyRedirectsGenerated:
 redirects:
 - /pem/latest/pem_admin/07_managing_pem_agent/
 - /pem/latest/pem_agent/03_managing_pem_agent/ 
+- /pem/latest/pem_online_help/04_toc_pem_features/16_pem_scheduled_jobs/
```
Comment (nidhibhammar) [inline product_docs/docs/pem/8/managing_pem_agent.mdx:11]: Why this redirect is in version 8. We don't have this topic in version 9?

File: product_docs/docs/pem/9/pem_connecting_to_database_server.mdx [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
 
-<div id="pem_connect_to_server" class="registered_link"></div>
-
 After defining a server connection, use the `Connect to Server` dialog to authenticate with a server and access the objects stored on the server. To access the dialog, right click on the server name in the PEM client tree control, and select `Connect Server` from the context menu.
 
-![PEM Connect to Server dialog](../images/connect_to_server.png)
-
 If prompted, provide authentication information for the selected server:
 
 > -   Use the `Password` field to provide the password of the user that is associated with the defined server.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/pem_connecting_to_database_server.mdx:15]:
```suggestion
-   Use the `Password` field to provide the user's password associated with the defined server.
```

File: product_docs/docs/pem/9/pem_connecting_to_database_server.mdx [hunk 3]
```diff
@@ -1,29 +1,26 @@
...
 > -   Check the box next to `Save Password` to instruct the server to save the password for future connections; if you save the password, you will not be prompted when reconnecting to the database server with this server definition.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/pem_connecting_to_database_server.mdx:16]:
```suggestion
-   Select the box next to `Save Password` to instruct the server to save the password for future connections. If you save the password, you are not prompted while reconnecting to the database server with this server definition.
```

File: product_docs/docs/pem/8/managing_pem_agent.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/managing_pem_agent.mdx:11]: Added to v9

File: product_docs/docs/pem/9/in_app_help.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/06_index_dialog/
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/07_primary_key_dialog/
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/08_rls_policy_dialog/
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/09_rule_dialog/
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/10_table_dialog/
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/11_trigger_dialog/
+- /pem/latest/pem_online_help/05_toc_pem_management_basics/12_modifying_tables/12_unique_constraint_dialog/
+---
+
+The in-app help provides comprehensive details about the PEM web interface, including the menu items, charts and graphs on the dashboards, and the options on the dialog boxes. The in-app help also includes instructions for creating databases objects.
```
Comment (drothery-edb) [inline product_docs/docs/pem/9/in_app_help.mdx:59]: Databases should be singular in all of these sections.
```suggestion
The in-app help provides comprehensive details about the PEM web interface, including the menu items, charts and graphs on the dashboards, and the options on the dialog boxes. The in-app help also includes instructions for creating database objects.
```

**Review by drothery-edb — DISMISSED** [review]

**Review by josh-heyer — CHANGES_REQUESTED** [review]
The following existing paths fail to load or redirect in this branch: 

```
/pem/latest/pem_online_help/01_toc_pem_getting_started/05_group_dialog/
/pem/latest/pem_online_help/01_toc_pem_getting_started/11_connect_error/
/pem/latest/pem_online_help/02_toc_pem_agent/04_pem_agent_ha/
/pem/latest/pem_online_help/03_toc_pem_client/02_pem_toolbar/
/pem/latest/pem_online_help/03_toc_pem_client/04_preferences/
/pem/latest/pem_online_help/03_toc_pem_client/05_keyboard_shortcuts/
/pem/latest/pem_online_help/03_toc_pem_client/06_search_objects/
/pem/latest/pem_online_help/04_toc_pem_features/
/pem/latest/pem_online_help/05_toc_pem_management_basics/10_managing_cluster_objects/
/pem/latest/pem_online_help/06_toc_pem_bart_management/07_bart_backup_dialog/
/pem/latest/pem_online_help/08_toc_pem_developer_tools/
/pem/latest/pem_online_help/08_toc_pem_developer_tools/01_debugger/
/pem/latest/pem_online_help/08_toc_pem_developer_tools/03_pem_interpreting_graphical_query/
/pem/latest/pem_online_help/08_toc_pem_developer_tools/04_editgrid/
/pem/latest/pem_online_help/08_toc_pem_developer_tools/04_editgrid/01_viewdata_filter/
/pem/latest/pem_online_help/10_pgagent/
/pem/latest/pem_online_help/10_pgagent/01_using_pgagent/
/pem/latest/pem_online_help/10_pgagent/02_pgagent_install/
/pem/latest/pem_online_help/10_pgagent/03_pgagent_jobs/
/pem/latest/pem_online_help/10_pgagent/04_pgagent-steps/
/pem/latest/pem_online_help/10_pgagent/05_pgagent-schedules/
/pem/latest/pem_online_help/11_appendices/
/pem/latest/pem_online_help/11_appendices/01_licence/
/pem/latest/pem_online_help/11_appendices/02_kerberos/
/pem/latest/pem_online_help/11_appendices/03_openssl/
/pem/latest/pem_online_help/11_appendices/04_snmp++/
/pem/latest/pem_online_help/11_appendices/05_jquery_table_sort/
```

To avoid 404s, please ensure these are redirected to something sensible.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**